packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/portal/PortalControllerIntegrationTests.java

```diff
@@ -78,4 +88,27 @@ public void givenPortalWhenWeSubscribeToPortalEventsThenCurrentStateOfThePortalI
                 .thenCancel()
                 .verify(Duration.ofSeconds(10));
     }
+
+    @Test
+    @DisplayName("Given an arbitrary semantic element, then we can create a portal on it")
```

Comment:
"Given INITIAL STATE, when ACTION IS PERFORMED, then EXPECTED RESULT"

Reply:
Done.